app/src/main/java/in/co/rajkumaar/amritarepo/aumsV2/activities/HomeActivity.java

```diff
@@ -42,9 +44,35 @@ protected void onCreate(Bundle savedInstanceState) {
 
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
-        name.setText(preferences.getString("name", "N/A"));
-        user_name.setText(preferences.getString("username", "N/A"));
-        e_mail.setText(preferences.getString("email", "N/A"));
+        Encryption enc = null;
+        String rmName = null;
+        String rmEmail = null;
+        String rmUsername = null;
+        try {
+            rmName = preferences.getString("name", "N/A");
```

Comment:
Please give the default value as empty string, so that N/A as such doesn’t get populated in the edittexts.